time/README.md

```diff
@@ -0,0 +1,25 @@
+# Moonbit/Core Time
+
+## Overview
+
+Package time provides functionality for measuring and manipulating time.
+
+The calendrical calculations always assume a ISO 8601 calendar, with no leap seconds.
+
+To create a datatime that represents the current time, you need to obtain the unix second and time zone offset from the [wasi](https://mooncakes.io/docs/#/peter-jerry-ye/wasi/) package (wasm-gc backend) or other FFI functions, and manually create a datetime.
+
+```
+// creates a UTC+8 fixed time zone.
+let zone = @time.fixed_zone("Asia/Shanghai", 8*60*60)
+
+// creates a ZonedDateTime from unix second and time zone.
+let date_time = @time.unix(1714227729L, nanosecond=1000, ~zone) // Ok(2024-04-27T22:22:09.000001+08:00[Asia/Shanghai])
+```
+
+## TODOs
+
+- Convert from/to RFC format string.
```

Comment:
I assume it is [rc9557](https://datatracker.ietf.org/doc/html/rfc9557). Corret me if I'm wrong.

Reply:
I believe that [rfc3339](https://datatracker.ietf.org/doc/html/rfc3339) is more widely used compared to rfc9557, which is a newer standard. We should also consider supporting [rfc2822](https://datatracker.ietf.org/doc/html/rfc2822) that is used in emails.